content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -0,0 +1,291 @@
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+* Você precisa ter um cluster Kubernetes que tenha apenas um nó, e a ferramenta de linha de comando
+{{< glossary_tooltip text="kubectl" term_id="kubectl" >}} configurada para se comunicar com seu cluster. Se você
+ainda não tem um cluster de um único nó, você pode criar um usando o
+[Minikube](https://minikube.sigs.k8s.io/docs/).
+
+* Familiarize-se com o material em
+[Volumes persistentes](/docs/concepts/storage/persistent-volumes/).
+
+<!-- steps -->
+
+## Criando um arquivo index.html no seu Nó
+
+Abra um shell no único Nó do seu cluster. A maneira de abrir um shell vai depender de como 
+você inicializou seu cluster. Por exemplo, se você estiver usando o Minikube, 
+você pode abrir um shell para o seu Nó digitando `minikube ssh`.
+
+No seu shell desse Nó, crie um diretótio `/mnt/data`:
+
+```shell
+# Assumindo que o seu Nó use "sudo" para executar comandos 
+# como superusuário
+sudo mkdir /mnt/data
+```
+
+
+No diretório `/mnt/data`, crie o arquivo `index.html`:
+
+```shell
+# Novamente assumindo que seu Nó use "sudo" para executar comandos
+# como superusuário
+sudo sh -c "echo 'Hello from Kubernetes storage' > /mnt/data/index.html"
+```
+
+{{< note >}}
+Se o seu Nó usa uma ferramenta para acesso como superusuário que não `sudo`, você pode
+geralmente fazer isso funcionar substituíndo `sudo` pelo nome da outra ferramenta.
+{{< /note >}}
+
+Teste se o arquivo `index.html` existe:
+
+```shell
+cat /mnt/data/index.html
+```
+
+A saída deve ser:
+```
+Hello from Kubernetes storage
+```
+
+Você agora pode fechar o shell do seu Nó.
+
+## Crie um Volume Persistente
+
+Neste exercício, você cria um Volume Persistente *hostPath*. O Kubernetes suporta
+`hostPath` para desenvolvimento e teste em um cluster com apenas um Nó. Um Volume Persistente 
+`hostPath` usa um arquivo ou diretório no Nó, para emular um armazenamento conectado pela rede.
+
+Em um cluster de produção, você não usaria `hostPath`. Em vez disso um administrador 
+de cluster provisionaria um recurso de rede, como um disco persistente do 
+`Google Compute Engine`, um NFS compartilhado, ou um volume do 
+`Amazon Elastic Block Store`. Administradores podem também usar [classes de armazenamento]
+(/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#storageclass-v1-storage)
+para incializar [provisionamento dinâmico]
+(https://kubernetes.io/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes).
+
+Aqui está o arquivo de configuração para o Volume Persistente `hostPath`:
+
+{{< codenew file="pods/storage/pv-volume.yaml" >}}
+
+O arquivo de configuração especifica que o volume está em `/mnt/data` do Nó do cluster. 
+A configuração também especifica um tamanho de 10 gibibytes e um modo de acesso 
+`ReadWriteOnce`, o que significa que o volume pode ser montado como leitura-escrita 
+pelo único Nó. Define o [nome da classe de armazenamento](/docs/concepts/storage/persistent-volumes/#class)
```

Comment:
This link can be updated to link to the Portuguese version of the page (may require a change on the anchor name).